elements/cerium.md

```diff
@@ -3,4 +3,9 @@ layout: page
 title: Cerium
 ---
 
-Barium is a chemical element with the symbol Ce and atomic number 58.
\ No newline at end of file
+Symbol : Ce<br/>
+Atomic number : 58<br/>
+
+
+
+Barium is a chemical element with the symbol Ce and atomic number 58.
```

Comment:
Should be Cerium instead of Barium?

Reply:
Yes, it should be Cerium instead of Barium. I am sorry for the typo. Shall I make the update and submit again?